### PR TITLE
XWIKI-16344: Use .xform style standard in the wiki and wysiwyg edit modes

### PR DESCRIPTION
**Pull request by Haxsen**
Issue link: https://jira.xwiki.org/browse/XWIKI-16344

Before:
Picture 2020-02-28 00_30_48

After:
Picture 2020-02-28 00_29_50

Please let me know any mistakes.
Thanks to 9inpachi for helping me on this issue!

### CODE REVIEW

**Comment by 9inpachi** [comment]
TL;DR - Use a single `<dl>` that is a parent to both the title and content and the content edit area as well.

Using a `<dl>` tag for "Content" is causing a margin just below "Content" as you might have noticed.
Can you find a way to use the same `<dl>` tag for both the "Title" and "Content"?

I did it in the console and got this:
image

I tried adding `<dl>` directly under `id="editMeta"` but that doesn't seem to solve the problem even if I make a single parent `<dl>` for both "Title" and "Content" because `<dl>` has a bottom margin. We should cover the whole content edit area in the `<dl>` as well so the margin comes below the content edit area where it is supposed to be.
So I think the `<dl>` would stretch over the content editor as well.

**Comment by 9inpachi** [comment]
I think the `<dl>` would work good directly inside the `.row` [here](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm#L181) or directly inside the `#mainEditArea` [here](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm#L173).

Hope that helps you. :)

**Comment by Haxsen** [comment]
Working on it, thanks!

**Comment by Haxsen** [comment]
Thanks, I thought this could be another issue. 
Can you check it now?

**Comment by 9inpachi** [comment]
Everything looks good now! :)
Again a great job with the PR! You show that you are starting to grasp things.

**Comment by Haxsen** [comment]
Tested it as well, screenshot:
Picture 2020-02-28 02_52_16

**Comment by 9inpachi** [comment]
A minor comment: For the future, please make the commit messages follow the [standard](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRule:AlwaysputaJIRAissuereferenceincommitmessages).

**Comment by Haxsen** [comment]
Thanks, got it.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm [hunk 1]
```diff
@@ -179,26 +179,28 @@
...
-      #if($editor == 'rights')
-        $services.localization.render('core.editors.rights.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'object')
-        $services.localization.render('core.editors.object.title', [$escapetool.xml($doc.plainTitle), $doc.getURL()])
-      #elseif($editor == 'class')
-        $services.localization.render('core.editors.class.title', [$escapetool.xml($doc.fullName), $doc.getURL()])
+    <dl>
+      #if ($displayContentMenu)
+        <div class="col-xs-12 col-md-5 pull-right">
+          #template("menus_content.vm")
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Is this template generating ``dt`` or ``dd`` elements? Otherwise the HTML is invalid. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl . You can use ``div`` inside a definition list, but the goal is to wrap ``dt`` and ``dd``.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm [hunk 2]
```diff
@@ -179,26 +179,28 @@
...
+        </div>
       #end
-      </h1></div>
-    #end
+      #if (($editor == 'wiki' || $editor == 'wysiwyg') && !$request.section)
+        <div id="editMeta" class="col-xs-12#if ($displayContentMenu) col-md-7#end">
+          #template('editmeta.vm')
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Same as above, this produces invalid HTML most probably.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm [hunk 3]
```diff
@@ -179,26 +179,28 @@
...
+        </div>
+      #else
+        <div id="document-title" class="col-xs-12#if ($displayContentMenu) col-md-7#end"><h1>
+        #if($editor == 'rights')
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Same, there's no ``dt`` or ``dd`` wrapped here, but a level 1 heading.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 4]
```diff
@@ -34,13 +34,19 @@
...
+    </dd>
 </div>
 #end
 
 #if($editor == 'wiki')
 <div id="contentMeta">
-  <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
+    <dt>
+      <label for="content">$services.localization.render('core.editors.content.contentField.label')</label>
+    </dt>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Where is the ``dd`` that corresponds to this definition term?

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 5]
```diff
@@ -34,13 +34,19 @@
 </div>
 #if ($editor != 'inline')
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+    <dt>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Inconsistent code indentation. The rest of the code uses 2 spaces.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 6]
```diff
@@ -34,13 +34,19 @@
...
+      <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
+    </dt>
+    <dd>
+      <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Line too long. I know you just copied the line from above, but since you're modifying it it would be nice to wrap before it reaches the 120 char limit.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 1 above)
Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: I don't understand why you mention ``editmeta.vm`` when the code above includes ``menus_content.vm`` which doesn't generate any ``dt`` or ``dd``. Do you see any example on the [Vertical Forms](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/) documentation where the ``dl`` has **direct** child nodes other than ``dt`` and ``dd``?

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 2 above)
Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: That's not what I see. I see this:

```
<dl>
  <div id="editMeta" class="col-xs-12 col-md-7">
    <div id="parentinput" class="form-group">
      <label for="xwikidocparentinput">...</label>
      <input type="text" id="xwikidocparentinput" name="parent"  ..."/>
    </div>
    ...
```

which is definitely not a valid definition list for me.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 3 above)
Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Where do you see on the [Vertical Form](https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/) documentation an example like this:

```
<dl>
  <div>
    <h1>...</h1>
```

Again, HTML validity is beyond the Vertical Form documentation, which is specific to XWiki. You cannot have vertical forms with invalid HTML.

**Comment by Haxsen** [comment]
@mflorea 
When I edit only the `editmeta.vm` (which I initially did), it becomes like this:
Picture 2020-03-02 23_44_24
Picture 2020-03-02 23_44_49

Which produces output like this:
Picture 2020-03-02 23_45_08

Which creates another issue, i.e. 'Content' label no more aligned vertically well.
And content's `dd` simply doesn't exist (originally) and it's separated from this code, i.e. It's in another `<div>` (see below screenshot)
Picture 2020-03-02 23_52_22

So, for the first issue (Content label not aligned) 9inpachi helped me and told me to use <dl> outside (in the parent file, i.e. `edit.vm`).
And when the parent file `edit` picks `editmeta` , the final code becomes a valid html like this:
```dl
##Title label
dt
/dt
dd
/dd
##now Content label
dt
/dt```

I still might not be understanding you well, just give me a little explanation on: How could I improve this ?
I don't have a better approach in my mind.
Would love to learn from you :)

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 4 above)
Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: A definition term (``dt``) should be followed by a definition (``dd``). If there's no definition then the ``dd`` remains empty.

**Comment by mflorea** [comment]
@Haxsen in order to progress you need read my comments carefully and reply to **each** of them individually. ATM there are 4 comments that you haven't addressed, one of which is a question that waits for an answer.

**Comment by Haxsen** [comment]
Ok @mflorea , give me some time.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 1 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: No there isn't direct child nodes to `dl` . Which I initially did. But it was creating another issue.
I'll see what I can do about this. But what about Content's `dd` (which is in another `div`) ?

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 4 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Ok got it.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 1 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: I can remove `dl` from here and add it around `#template('editmeta.vm')` but it will still be an invalid HTML. I can only use `dl` inside `editmeta.vm` to make a valid HTML. 
But again, content label gets unaligned.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 3 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Um, yes that's true. So I can not have `dl` in main file `edit.vm` , that's what I learned.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 2 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Hm, so I have to use `dl` in `edit.vm` only.

**Comment by Haxsen** [comment]
@mflorea 
I have replied on your comments.
Summary: I have to use `dl` where the labels are (editmeta.vm) and no direct child nodes.
And content's `dd` will be empty (`<dd>  </dd>`).

**Comment by Haxsen** [comment]
@mflorea 
Now the HTML is valid.
Please check the files and let me know the problems.
Thanks for the help!

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm (hunk 1 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: Anyways, I removed `dl` from here since it was not wrapping for a valid HTML in this file.

Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/edit.vm]: You removed the ``dl`` but the code still appears as modified because you left the indentation. Make sure you have only relevant changes in the diff, otherwise it's hard to review.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 7]
```diff
@@ -29,18 +29,31 @@
 ## Title and parent
 ##
 <div id="parentinput" class="form-group">
-  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
+  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') 
+  <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: The ``a`` element is not on the same level as the ``label``. In fact it is a child element, and child elements should be indented.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 8]
```diff
@@ -29,18 +29,31 @@
...
 <div id="titleinput" class="form-group">
-  <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
-  <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
+  <dl>
+    <dt>
+      <label for="xwikidoctitleinput">$services.localization.render('core.editors.content.titleField.label')</label>
+    </dt>
+    <dd>
+      <input type="text" id="xwikidoctitleinput" name="title" value="$escapetool.xml("$!docTitle")" 
+      class="#if($xwiki.getXWikiPreference('xwiki.title.mandatory') == 1)required#end"/>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: It's good that you wrap the long line, but you also need to indent, otherwise the code is hard to read.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 7 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Alright, sorry I didn't notice!

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 8 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Got it!

Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: And also I mistakenly requested re-review when I saw you did not respond (but you did and my browser had loaded a cache file where I couldn't see, so I refreshed and noticed).

**Comment by Haxsen** [comment]
Now check it @mflorea  :)
Let me know if still there are problems!

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 9]
```diff
@@ -29,18 +29,31 @@
 ## Title and parent
 ##
 <div id="parentinput" class="form-group">
-  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
+  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') 
+    <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: This line still has more than 120 characters.

**Comment by Haxsen** [comment]
Screenshots are same as in the original post, you can see the space after `Content` label.

**Comment by mflorea** [comment]
Maybe we can fix the space from CSS.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 9 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Done, also I found another line in `edit.vm` (check my commit) . It was #velocity code, so is it fine or have I done it wrong? Check my commits to see.

**Comment by Haxsen** [comment]
Yes maybe it's caused by `dl`, let me show you first:
Picture 2020-03-05 21_25_17

**Comment by Haxsen** [comment]
Ok, now I changed `dl` style:
Picture 2020-03-05 21_33_34

And problem seems to be solved, but I am afraid that it might cause this change to all `dl` where it's used (.xform).

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm [hunk 10]
```diff
@@ -29,18 +29,32 @@
 ## Title and parent
 ##
 <div id="parentinput" class="form-group">
-  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') <a id='hideEditParentTrigger' href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
+  <label for="xwikidocparentinput">$services.localization.render('core.editors.content.parentField.label') 
+    <a id='hideEditParentTrigger' 
+     href='#'>$services.localization.render('core.editors.content.parentField.edit.hide')</a></label>
```
Comment (mflorea) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: That's not he way you should format HTML. Why not simply:

```
<label ...>
  <a ...>
    ...
  </a>
</label>
```

But this formatting is out of the scope of this PR. It's probably better to keep only the changes that are relevant for the issue you're trying to fix.

**Comment by mflorea** [comment]
Changing the styles for all definition lists is obviously not an option. The idea is to fix this particular definition list.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 10 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: But it is over 120 chars, should I format it or keep it as is?

Comment (mflorea) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Looks good, but I would prefer to keep in this PR only the changes that are relevant for the current issue. You can make later another PR to fix the formatting.

**Comment by Haxsen** [comment]
Can you give me a hint on how can I apply this fix for this particular definition list?
I think I will have to create a specific style of `dl` for this content label. I am not sure how to.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm (hunk 10 above)
Comment (Haxsen) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editmeta.vm]: Ok, let me do this quickly.

**Comment by mflorea** [comment]
If you're asking how to apply custom styles to a specific element then you need to read about CSS selectors. If you're asking where to put these custom styles then you should check if the elements near this definition list have custom styles and if they have then find the place where those are written and add there the custom styles for this particular definition list.

**Comment by Haxsen** [comment]
Thanks, I am closing this PR and will create new one(s) for this issue and formatting/indentation.
